/jds/bin/diff -uprN gnome-control-center-2.25.90.old/capplets/default-applications/default-applications.desktop.in.in gnome-control-center-2.25.90/capplets/default-applications/default-applications.desktop.in.in
--- gnome-control-center-2.25.90.old/capplets/default-applications/default-applications.desktop.in.in	2009-02-10 18:48:32.461183000 +0000
+++ gnome-control-center-2.25.90/capplets/default-applications/default-applications.desktop.in.in	2009-02-10 18:49:10.366371000 +0000
@@ -1,6 +1,7 @@
 [Desktop Entry]
 _Name=Preferred Applications
-_Comment=Select your default applications
+# SUN_BRANDING
+_Comment=Choose applications to open different types of documents
 Exec=gnome-default-applications-properties
 Icon=gnome-settings-default-applications
 Terminal=false
/jds/bin/diff -uprN gnome-control-center-2.25.90.old/capplets/keyboard/keyboard.desktop.in.in gnome-control-center-2.25.90/capplets/keyboard/keyboard.desktop.in.in
--- gnome-control-center-2.25.90.old/capplets/keyboard/keyboard.desktop.in.in	2009-02-10 18:48:32.335953000 +0000
+++ gnome-control-center-2.25.90/capplets/keyboard/keyboard.desktop.in.in	2009-02-10 18:50:24.271715000 +0000
@@ -1,6 +1,7 @@
 [Desktop Entry]
 _Name=Keyboard
-_Comment=Set your keyboard preferences
+# SUN_BRANDING
+_Comment=Set auto-repeat preferences and enforce typing breaks
 Exec=gnome-keyboard-properties
 Icon=preferences-desktop-keyboard
 Terminal=false
/jds/bin/diff -uprN gnome-control-center-2.25.90.old/capplets/mouse/gnome-settings-mouse.desktop.in.in gnome-control-center-2.25.90/capplets/mouse/gnome-settings-mouse.desktop.in.in
--- gnome-control-center-2.25.90.old/capplets/mouse/gnome-settings-mouse.desktop.in.in	2009-02-10 18:48:32.402373000 +0000
+++ gnome-control-center-2.25.90/capplets/mouse/gnome-settings-mouse.desktop.in.in	2009-02-10 18:51:13.209387000 +0000
@@ -1,6 +1,7 @@
 [Desktop Entry]
 _Name=Mouse
-_Comment=Set your mouse preferences
+# SUN_BRANDING
+_Comment=Set mouse button and motion preferences
 Exec=gnome-mouse-properties
 Icon=input-mouse
 Terminal=false
/jds/bin/diff -uprN gnome-control-center-2.25.90.old/capplets/network/gnome-network-properties.desktop.in.in gnome-control-center-2.25.90/capplets/network/gnome-network-properties.desktop.in.in
--- gnome-control-center-2.25.90.old/capplets/network/gnome-network-properties.desktop.in.in	2009-02-10 18:48:32.344460000 +0000
+++ gnome-control-center-2.25.90/capplets/network/gnome-network-properties.desktop.in.in	2009-02-10 18:51:33.105975000 +0000
@@ -1,6 +1,7 @@
 [Desktop Entry]
 _Name=Network Proxy
-_Comment=Set your network proxy preferences
+# SUN_BRANDING
+_Comment=Set up a network proxy for most desktop applications
 Exec=gnome-network-properties
 Icon=gnome-network-properties
 Terminal=false
/jds/bin/diff -uprN gnome-control-center-2.25.90.old/capplets/windows/window-properties.desktop.in.in gnome-control-center-2.25.90/capplets/windows/window-properties.desktop.in.in
--- gnome-control-center-2.25.90.old/capplets/windows/window-properties.desktop.in.in	2009-02-10 18:48:32.427337000 +0000
+++ gnome-control-center-2.25.90/capplets/windows/window-properties.desktop.in.in	2009-02-10 18:52:05.762481000 +0000
@@ -1,6 +1,7 @@
 [Desktop Entry]
 _Name=Windows
-_Comment=Set your window properties
+# SUN_BRANDING
+_Comment=Change window selection, movement and sizing behavior
 Exec=gnome-window-properties
 Icon=preferences-system-windows
 Terminal=false
diff -ruN gnome-control-center-2.29.92.orig/capplets/display/display-properties.desktop.in.in gnome-control-center-2.29.92/capplets/display/display-properties.desktop.in.in
--- gnome-control-center-2.29.92.orig/capplets/display/display-properties.desktop.in.in	2010-03-12 01:50:34.783873462 +0000
+++ gnome-control-center-2.29.92/capplets/display/display-properties.desktop.in.in	2010-03-12 01:50:54.011269760 +0000
@@ -1,6 +1,7 @@
 [Desktop Entry]
 _Name=Monitors
-_Comment=Change resolution and position of monitors
+# SUN_BRANDING
+_Comment=Adjust the screen resolution and rotation
 Exec=gnome-display-properties
 Icon=gnome-display-properties
 Terminal=false
